app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -32,22 +32,25 @@
 				"bureau_de_change",	"money_transfer", "post_office", "library",
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
-				"doctors", "clinic", "pharmacy", "veterinary" };
+				"doctors", "clinic", "pharmacy", "veterinary",
+				"place_of_worship", "school", "townhall"};
```

Comment:
You have a point there. However, while a school may not as private as I depicted, it is surely much less public than any other place mentioned in the list of places for which the quest shows a marker.

If there is any doubt or uncertainty regarding whether an element should be included or not, I rather have it not in the app. That is why the app does not ask for surface of service roads and trunks, it does not ask about the roof shape of industrial buildings etc. etc. Basically for almost every quest, the list of elements for which it is shown is smaller than it could be, because there is always a grey zone where "probably" it would still make sense for some/many/most elements of a certain type. I'd like the app to be as white as possible in that regards. There is enough else to do, no need to exhaust every quest type to the max - this will only contribute to the perception of some OpenStreetMappers that SC is a QA tool that shows all sorts of ("can but don't have to") false positives.